code-challenges.test.js

```diff
@@ -26,42 +26,110 @@ const numbersArray2 = [24, 27, 30, 33, 36]
-
-const object1 = { number: 15 }
-// Expected output: "15 is divisible by three"
-const object2 = { number: 0 }
-// Expected output: "0 is divisible by three"
-const object3 = { number: -7 }
-// Expected output: "-7 is not divisible by three"
-
+// describe("divisibleBy3", () => {
+//     it("Takes in an object and returns if divisible by three.", () => {
+//       const object1 = { number: 15 }
+//       const object2 = { number: 0 }
+//       const object3 = { number: -7 }
+//       expect(divisibleBy3(object1.number)).toEqual("is divisible by three")
+//       expect(divisibleBy3(object2.number)).toEqual("is divisible by three")
+//       expect(divisibleBy3(object3.number)).toEqual("is not divisible by three")
+//     })
+//   }) //● divisibleBy3 › Takes in an object and returns if divisible by three.
+  
 // b) Create the function that makes the test pass.
-
+const divisibleBy3 = (object) => {
+      if (object % 3 === 0){
+        return "is divisible by three"
+      } else if (object % 3 !== 0){
+        return "is not divisible by three"
+      }
+}  
 // Pseudo code:
+// Input: Object
+// Output: "if the Number is divible by 3"
+// Create a function named divisibleBy3
+// takes in an obect key value which will be a number as an argument
+// crate a conditional if/ else if statement 
+// with the modulo % check if each argument is divisible by three. 
+// return a string saying if it is or not
 
 // --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
 
 // a) Create a test with expect statements for each of the variables provided.
-
-const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
-// Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
-const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
-// Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
-
+// describe("capArray", () => {
+//     it("takes in an array of words and returns an array with all the words capitalized.", () => {
+//         const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+//         const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
+//       expect(capArray(randomNouns1)).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
+//       expect(capArray(randomNouns2)).toEqual(["Temperature", "Database", "Chopsticks", "Mango"])
+//     })
+//   }) //capArray › takes in an array of words and returns an array with all the words capitalized.
 // b) Create the function that makes the test pass.
-
+const capArray = (array) => {
+  let newArray = [];
+  for (let i = 0; i < array.length; i++) {
+    newArray.push(array[i].charAt(0).toUpperCase());
+  }
+  return newArray;
+}; // i got lost trying to find the issue. later i added .charAt method make sure it was the first lleter capitalized. Array [
```

Comment:
The console here is showing you what it expected (lines 76 - 80) and then showing you what it received instead (lines 81 - 85). So as it currently is, the function is only returning the first letter of each word capitalized. In your function, each iteration in the for loop is pushing something into the newArray. However, you only told it to push the element at index 0 upper cased, but forgot about the rest of the word.